app/assets/javascripts/availability.js.coffee

```diff
@@ -13,7 +13,8 @@ class AvailabilityUpdater
     'Remote Storage Request', 'Hold Request', 'Recall Request']
   missing_statuses = ['Missing', 'Lost--Library Applied',
```

Comment:
Is it worth pulling these statuses from a config file rather than hardcoding them?